scanner/src/main/kotlin/scanners/fossid/FossIdScanResults.kt

```diff
@@ -83,7 +83,13 @@ internal fun <T : Summarizable> List<T>.mapSummary(
 
         summary.licences.forEach {
             runCatching {
-                LicenseFinding.createAndMap(it.identifier, location, detectedLicenseMapping = detectedLicenseMapping)
+                DeclaredLicenseProcessor.process(it.identifier)?.let { spdx ->
```

Comment:
Given our latest thoughts, it really correct to use a mapping of _declared_ licenses here, *and* in the fallback a mapping of _detected_ licenses?

If the line of thinking is that we only get detected licenses here, then we should only apply `detectedLicenseMapping` here.

Or phrased differently, the issue you're mentioning in the commit message would already be solved with the changes in `mapSnippetFindings()` alone, correct?

Reply:
Yes you're right. The issue I mentioned only needs the fix in `mapSnippetFindings()`.